core/src/test/java/org/springframework/security/authentication/UserDetailsRepositoryReactiveAuthenticationManagerTests.java

```diff
@@ -56,6 +55,9 @@
 	@Mock
 	private Scheduler scheduler;
 
+	@Mock
+	private UserDetailsChecker userDetailsChecker;
```

Comment:
Can you please rename this to `postAuthenticationChecks` to be consistent?

Reply:
Done!